x-pack/solutions/observability/plugins/infra/public/components/kubernetes_dashboard_promotion/kubernetes_dashboard_promotion.tsx

```diff
@@ -0,0 +1,318 @@
+        url: string;
+        text: string;
+        testSubj: string;
+      };
+    };
+    actionButton: {
+      installed: {
+        dashboardsPath: string;
+        label: string;
+        testSubj: string;
+      };
+      notInstalled: {
+        integrationPath: string;
+        label: string;
+        testSubj: string;
+      };
+    };
+    hideButton: {
+      label: string;
+      ariaLabel: string;
+      testSubj: string;
+    };
+  };
+  link: {
+    dashboardsPath: string;
+    label: string;
+    testSubj: string;
+  };
+}
+
+const ECS_CONFIG: IntegrationConfig = {
+  card: {
+    imageType: 'ecs',
+    title: {
+      installed: i18n.translate(
+        'xpack.infra.kubernetesDashboardPromotion.ecs.integrationInstalledTitle',
+        { defaultMessage: 'View Kubernetes Dashboards' }
+      ),
+      notInstalled: i18n.translate(
+        'xpack.infra.kubernetesDashboardPromotion.ecs.integrationNotInstalledTitle',
+        { defaultMessage: 'Install Kubernetes Integration' }
+      ),
+    },
+    description: {
+      installed: (integrationLink: React.ReactNode) => (
+        <FormattedMessage
+          id="xpack.infra.kubernetesDashboardPromotion.ecs.integrationInstalledDescription"
+          defaultMessage="View dashboards available for the Kubernetes clusters observed using the {integrationLink} which match your query."
+          values={{ integrationLink }}
+        />
+      ),
+      notInstalled: (integrationLink: React.ReactNode) => (
+        <FormattedMessage
+          id="xpack.infra.kubernetesDashboardPromotion.ecs.integrationNotInstalledDescription"
+          defaultMessage="We have dashboards for Kubernetes clusters observed using the {integrationLink} which match your query."
+          values={{ integrationLink }}
+        />
+      ),
+      docsLink: {
+        url: 'https://www.elastic.co/docs/reference/integrations/kubernetes',
+        text: i18n.translate(
+          'xpack.infra.kubernetesDashboardPromotion.ecs.integrationDocsLinkText',
+          {
+            defaultMessage: 'Kubernetes Integration',
+          }
+        ),
+        testSubj: 'infraKubernetesDashboardCardIntegrationDocsLink',
+      },
+    },
+    actionButton: {
+      installed: {
+        dashboardsPath: '#/list?_g=()&s=tag:(Kubernetes)',
+        label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.viewDashboardsLink', {
+          defaultMessage: 'View Dashboards',
+        }),
+        testSubj: 'infraKubernetesDashboardCardLink',
+      },
+      notInstalled: {
+        integrationPath: '/detail/kubernetes',
+        label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.viewIntegrationLink', {
+          defaultMessage: 'View Integration',
+        }),
+        testSubj: 'infraKubernetesDashboardCardInstallLink',
+      },
+    },
+    hideButton: {
+      label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.hideCard', {
+        defaultMessage: 'Hide this',
```

Comment:
Is there a reason why the text on the button and the aria-label are different? Can we not show `Dismiss` as we have for other call-outs and use same i18n label